src/extensions/default/bramble/lib/iframe-browser.js

```diff
@@ -32,7 +32,7 @@ define(function (require, exports, module) {
 
         // If iframe does not exist, then show it
         if(result.rows === 1 && result.columns === 1) {
-            show(_orientation);
+            setLayout(_orientation);
```

Comment:
Here you're assuming that `setLayout` accepts an `orientation` argument, but below in line 71, you don't use it.  We need to sort that out.

Reply:
good catch...looks like a bug that Brackets had initially. I think just removing the argument passed in here should be good enough since `_orientation` is global to this module.